src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -139,12 +150,45 @@ export const WorkspaceDashboard = _.flow(
           case 404:
             this.setState({ consentStatus: `Structured Data Use Limitations are not available for ${orspId}` })
             break
-          default: throw error
+          default:
+            throw error
         }
       }
     }
   })
 
+  getTagSuggestions = debouncePromise(withErrorReporting('Error loading tags', async text => {
+    const { ajax: { Workspaces } } = this.props
+    if (text.length > 2) {
+      return _.map(({ tag, count }) => {
```

Comment:
Can we also filter out tags that have already been applied to this workspace?

Reply:
I'm going to make a ticket for that as a follow on (will insert ticket number here shortly)